### PR TITLE
Consistent header and footer for blog

### PR DESCRIPTION
**Pull request by mosabua**
- externalized header and footer from default
- added usage for blog related layouts
- added youtube channel link to footer (and image)
- added highlighting for blog link in nav

Signed-off-by: Manfred Moser <mmoser@apache.org>
Signed-off-by: Manfred Moser <manfred@simpligility.com>

### CODE REVIEW

**Comment by mosabua** [comment]
Screen Shot 2019-11-04 at 14 04 08

preview image from local run of site

**Comment by mosabua** [comment]
@electrum @martint @dain @findepi  .. feel free to have a look and merge away.

File: _layouts/blog.html [hunk 1]
```diff
@@ -5,21 +5,19 @@
...
-        <div class="trigger">
-          <a class="page-link" href="{{ '/' | relative_url }}">About Presto</a>
-        </div>
-      </nav>
-    </div>
+  {%- include header.html -%}
+
+  <header class="hero clearfix">
+      <div class="width">
+          <h1>Blog</h1>
```
[the remark this answers is not in the document]
Comment (mosabua) [reply _layouts/blog.html]: hm ... probably a side effect of the separate css for blog .. let me debug this some more

[the remark this answers is not in the document]
Comment (mosabua) [reply _layouts/blog.html]: Ouch ... side effects galore.

[the remark this answers is not in the document]
Comment (mosabua) [reply _layouts/blog.html]: Found and pushed a fix. Reordering css imports ftw!

**Comment by martint** [comment]
There's something about the website css that's causing the layout in at least one blog post to break. Take a look at:

https://deploy-preview-53--prestosql.netlify.com/blog/2019/05/21/optimizing-the-casts-away.html#putting-it-all-together
https://prestosql.io/blog/2019/05/21/optimizing-the-casts-away.html#putting-it-all-together

See how the formula numbers on the right no longer line up.

**Comment by mosabua** [comment]
Also made it so that the width of the blog matches the width of the other pages on the site now. Can we get this merged?

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**